src/storybook/components/text-colors/text-colors.jsx

```diff
@@ -17,3 +17,5 @@ export const TextColors = () => {
   );
   return <ColorsDescription colorNames={colorKeys} />;
 };
+
```

Comment:
Maybe we will want to exprot the colors descriptions for other storybook colors docs but I think there is no need to export our table of text colors

Reply:
Removed from `published-storybook-components`